### PR TITLE
Update OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by wuhuizuo**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
   sig-approvers: # default approvers for the repo on rest folders.
     - cfzjywxk
     - lance6716
     - overvenus
     - you06
     - YuJuncen
   sig-approvers-pb: # default approvers for `proto` folder
-    - kevin-xialiu
     - yudongusa
     - zhangjinpeng1987
```
Comment (wuhuizuo) [inline OWNERS_ALIASES:14]:
```suggestion
    - zhangjinpeng87
```

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
     - cfzjywxk
     - lance6716
     - overvenus
     - you06
     - YuJuncen
   sig-approvers-pb: # default approvers for `proto` folder
-    - kevin-xialiu
     - yudongusa
-    - zhangjinpeng1987
+    - zhangjinpeng87
```
Comment (wuhuizuo) [inline OWNERS_ALIASES:14]: fix github id

**Review by cfzjywxk — APPROVED** [review]

**Review by zhangjinpeng87 — APPROVED** [review]
LGTM

**Review by YuJuncen — APPROVED** [review]

**Pull request merged**